cppapi/server/jpeg/jpeg_bitstream.h

```diff
@@ -84,7 +84,6 @@ class OutputBitStream {
 #endif
    unsigned char *bufferPtr;
    unsigned char *numbits;
-   unsigned char  bScratch[4];
```

Comment:
It looks like this bScratch field is used when JPG_USE_ASM_PB macro is defined. 
This macro seems to be linked to --enable-jpegmmx/--disable-jpgmmx options given to configure script of the Tango source distribution.
So, we should probably surround bScratch with some
#ifdef JPG_USE_ASM_PB directives

Reply:
@bourtemb Done. This was also the reason why the 32bit appveyor tests failed.